modules/reference/pages/node-properties.adoc

```diff
@@ -206,7 +206,9 @@ List of seed servers used to join an existing cluster.
 
 If a cluster does not already exist:
 
-* When <<empty_seed_starts_cluster,empty_seed_starts_cluster>> is `true`, if the seed_servers list is empty, this broker will be the cluster root to form a new cluster that other brokers subsequently join. Exactly one broker in the cluster should set seed_servers to be empty when first bootstrapping a cluster. For brokers that are not the root broker, this is the list of brokers used to join the cluster.
-* Otherwise, when <<empty_seed_starts_cluster,empty_seed_starts_cluster>> is `false`, this refers to the list of brokers that initially bootstrap the cluster. In this case, seed_servers cannot be empty, and seed_servers must be identical for all brokers in that list. For brokers not in the seed_servers list, this is the list of brokers used to join the cluster.
+* A broker with an <<empty_seed_starts_cluster,empty_seed_starts_cluster>> set to `true` and an empty `seed_servers` list becomes the cluster root to form a new cluster for other brokers to join. Set `seed_servers` to empty for exactly one broker when first bootstrapping a cluster. 
+* Otherwise, when <<empty_seed_starts_cluster,empty_seed_starts_cluster>> is `false`, `seed_servers` is the list of brokers that initially bootstrap the cluster. In this case, `seed_servers` cannot be empty and must be identical for all brokers in that list. 
+
+For brokers that are not the root broker or are not in the `seed_servers` list, `seed_servers` is the list of brokers used to join the cluster.
```

Comment:
```suggestion
* When `empty_seed_starts_cluster` is `true`, Redpanda enables one broker with an empty `seed_servers` list to initiate a new cluster. The broker with an empty `seed_servers` becomes the cluster root, to which other brokers must connect to join the cluster.  Brokers looking to join the cluster should have their `seed_servers` populated with the cluster root's address, facilitating their connection to the cluster.
+
[IMPORTANT]
====
Only one broker, the designated cluster root, should have an empty `seed_servers` list during the initial cluster bootstrapping. This ensures a single initiation point for cluster formation.
====

* When `empty_seed_starts_cluster` is `false`, Redpanda requires all brokers to start with a known set of brokers listed in `seed_servers`. The `seed_servers` list must not be empty and should be identical across these initial seed brokers, containing the addresses of all seed brokers. Brokers not included in the `seed_servers` list use it to discover and join the cluster, allowing for expansion beyond the foundational members.
+
[NOTE]
====
The `seed_servers` list must be consistent across all seed brokers to prevent cluster fragmentation and ensure stable cluster formation.
====
```